grpc-api/grpc-api.gradle

```diff
@@ -51,7 +51,7 @@ dependencies {
     testCompile TestTools.projectDependency(project, 'DB'),
                 TestTools.projectDependency(project, 'Util')
 
-    runtimeOnly project(':Numerics'), project(':Integrations')
+    runtimeOnly project(':Numerics'), project(':Integrations'), project(':KafkaIngester')
```

Comment:
Why does our API depend on Kafka?

Reply:
grpc-api is mis-named.  This is essentially the worker code; Numerics and Integrations and Plot are here so that the worker can implement them.